src/index.html

```diff
@@ -1,13 +1,435 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link rel="shortcut icon" type="image/ico"
+    href="./images/favicon_bose.ico">
+    <title>Bose</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin = "anonymous">
```

Comment:
You need to check and fix it everywhere
<img width="1073" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/64750958/46aabcf6-77fa-4f6e-ae3d-7e0992b027a9">